recommender/engines/ncf/training/pipeline.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=line-too-long, too-few-public-methods, useless-super-delegation
+# pylint: disable=fixme
+
+"""Neural Collaborative Filtering Pipeline"""
+
+from recommender.engine.preprocessing.embedder import (
+    Embedder,
+)  # TODO: import embedder from proper module
+from recommender.engines.base.base_pipeline import BasePipeline
+from recommender.engines.ncf.training.data_extraction_step.data_extraction_step import (
```

Comment:
we could leverage the steps to __init__ in ...ncf.training, the imports are unnecessarily convoluted now

Reply:
okey, could be easily done even now.